### ISSUE TITLE
"System" colour palette isn't affected by Windows 10 "dark mode"

### ISSUE THREAD

**Issue by solemnwarning** [solemnwarning/rehex#101]
Reported by @hogsy. I'm guessing the shiny new _dark mode_ does its own special thing rather than using the theme APIs that have existed since Windows 95. Its the Windows 10 way.

**Comment by learn-more** [comment]
Yeah, this seems to mostly be an UWP thing, and wx is built on top of 'classic' win32 controls:
https://forums.wxwidgets.org/viewtopic.php?t=45869

**Comment by learn-more** [comment]
This is what the 'dark' color scheme looks like:

![image](https://user-images.githubusercontent.com/610685/98919496-2acf7000-24cf-11eb-9c07-29fe151d5c52.png)

**Comment by solemnwarning** [comment]
Well making a serious attempt at fixing this has been a horrible road of discovery.

So, there have been various (undocumented) hacks added to win32 to allow apps like explorer to have dark mode despite using the traditional UI elements, the most complete example of how to use them I found is:

https://github.com/ysc3839/win32-darkmode

...but I couldn't get that code working as part of rehex - the thing remains stubbornly bright no matter what I do.

I also found this horrible little gem of mintty trying to keep up with this comedy routine:

https://github.com/mintty/mintty/issues/983

On top of that, while fucking around with this I found other win32 apps (like Paint and Notepad) doesn't even fucking respect Dark Mode... because Microsoft themselves can't even keep up with this crap. And then @hogsy pointed out Windows 10 has an accent colour option... which also doesn't seem to affect the older colour palette because why would it.

I give up. I'm not going to waste any more time trying to support this particular bin fire. If anyone wants Dark Mode to work properly patches are welcome... but they'd probably be best sent to the wxWidgets project so they work for everyone.
